recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,49 @@
+c_compiler:                    # [win]
+  - vs2008                     # [win]
+  - vs2015                     # [win]
```

Comment:
@msarahan do we have vs2010 (vc10 for Python 3.4 available on `c3i`? Asking for a friend :smile:)

Reply:
Sort of.  https://github.com/AnacondaRecipes/aggregate/tree/master/vs2010

This does not account for Appveyor's windows SDK version of VS 2010.  Only the "real" VS2010.  I have not had time to port over the special casing for the other.  Same story with VS2008.